publications/medium-2015-05-15.md

```diff
@@ -0,0 +1,28 @@
+### io.js Pekan 15 Mei
+Minggu ini tidak ada perilisan io.js.
+
+### Berita io.js
+
+Dengan bangga kami mengumumkan berita ini, **io.js telah memutuskan untuk bergabung dengan Node Foundation**. Dan nama organisasi kami akan berubah menjadi `nodejs`.
+
+Sebagai langkah pertama, kami akan beralih dari organisasi `iojs` ke organisasi `nodejs` dan akan menyatu dengan `joyent/node` secara bertahap. Kami akan tetap merilis `io.js` sampai proses penyatuan selesai.
+
+Untuk melihat rencana penggabungan secara detail, silakan cek [jasnell repo](https://github.com/jasnell/dev-policy/blob/master/convergence.md). Jika Anda ingin menanyakan sesuatu, silakan bertanya pada [@Fishrock123](mailto:fishrock123@rocketmail.com). Arsip diskusi ada di [sini](https://github.com/nodejs/node/issues/1664). 
+
+
+### Update Komunitas
+
+* [Performance Showdown](https://raygun.io/blog/2015/05/performance-showdown-node-js-vs-io-js-v2-0-0/): Node.js vs. io.js v2.0.0.
+* Microsoft akan mengusulkan Node untuk menjalankan JavaScript _backend_ yang berbeda termasuk `Chakra`.
+* [Grafik _benchmark_] {node,io}.js (http://thlorenz.com/benchgraph/).
+* [Anggota Komite Teknis](https://twitter.com/rvagg/status/598605393636429825) memutuskan untuk memindahkan io.js ke Node Foundation dan nama organisasi akan berubah menjadi nodejs.
+* Pertemuan Perangkat Keras WG di [YouTube](https://www.youtube.com/watch?v=EFJVMGRtDWE).
+* Pertemuan Penyatuan `io.js/node.js` di [YouTube](https://www.youtube.com/watch?v=8dxkM9vHmrY).
+* Pertemuan Komite Teknis di [YouTube](https://www.youtube.com/watch?v=UbYiFLf7MpU).
+
+### Acara Mendatang
+
+* Tiket [NodeConf Adventure](http://nodeconf.com/) sedang dijual, 11 - 14th Juni di Walker Creek Ranch, CA
+* Tiket [CascadiaJS](http://2015.cascadiajs.com/) sedang dijual, 8 - 10 Juli di Washington State
+* Tiket [BrazilJS Conf](http://braziljs.com.br/) sedang dijual, 21 - 22 Agustus di Shopping Center BarraShoppingSul
+* Tiket [NodeConf EU](http://nodeconf.eu/) sedang dijual, 6 - 9 September di Waterford, Ireland
```

Comment:
Satu lagi mas, tambahkan line baru di ujung dokumen. [Konvensi UNIX](https://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline).